### PR TITLE
Better error handling when failing to open CCID context

### PR DESCRIPTION
**Pull request by fdennis**
Solves the issue when Smart Card Service on Windows is disabled and ykman qt could not open at all.

Also fixed a typo.

### CODE REVIEW

File: ykman/descriptor.py [hunk 1]
```diff
@@ -153,9 +154,13 @@ def get_descriptors():
...
-        for dev in open_ccid():
-            if dev:
-                yield dev
+        try:
+            for dev in open_ccid():
+                if dev:
+                    yield dev
+        except smartcard.pcsc.PCSCExceptions.EstablishContextException:
+            logger.debug('Failed to establish CCID context. '
+                         'Is the pcscd service running?')
```
Comment (dagheyman) [inline ykman/descriptor.py]: maybe we should say: `Is pcscd/smart card services running?` since it's not called pcscd on windows.

**Comment by dagheyman** [comment]
Nice! Looks good, just needs some manual testing before merging.

**Comment by dagheyman** [comment]
@emlun Do you know why the tests are failing? Seems unrelated. Python3 only syntax?

**Comment by emlun** [comment]
Uh, that's a syntax error in the `mock` dependency, and it looks like a beta release? But yes, looks like py2-incompatible syntax. Do we care enough to restrict the dependency version or do we just drop py2 now?

**Comment by dagheyman** [comment]
I think restricting the dependency version is the correct fix for now.

**Pull request merged**